tests/jsTests/validationSpec.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Suite : validateNumber
+ * Description : Test suite to validate comrades'phone number
+ */
+describe('validateNumber', function() {
+    it('Should be phone number', function() {
+        expect(validation.isPhoneNumber('342345235')).toBeTruthy();
+    });
+
+    it('Should not be phone number', function() {
+        expect(validation.isPhoneNumber('324fdgfd215235')).not.toBeTruthy();
+    });
+});
+
+/**
+ * Suite : validateEmail
+ * Description : Test suite to validate users' email address
+ */
+describe('validateEmail', function() {
+    it('Should be email', function() {
+        expect(validation.isEmailAddress('abc@def.com')).toBeTruthy();
+    });
+
+	it('Should be email', function() {
+        expect(validation.isEmailAddress('mail.google.com')).not.toBeTruthy();

```

Comment:
I expect, that this would form a part of the module later. Why do you want to do things twice, if you finish this completely then you can directly plugin this in the tests. 

Even otherwise I do not have any issues, we can go ahead, but while the actual testing we would have to redo this more efficiently.